Classes/KIFUIViewTestActor.m

```diff
@@ -263,9 +283,10 @@ - (void)enterTextIntoCurrentFirstResponder:(NSString *)text fallbackView:(UIView
 - (void)setText:(NSString *)text;
 {
     KIFUIObject *found = [self _predicateSearchWithRequiresMatch:YES mustBeTappable:NO];
```

Comment:
Can *found be nil in this case and if so is it safe to use?
Getting rusty in obj c these days...